src/applications/vaos/appointment-list/components/PastAppointmentsListV2/PastAppointmentsDateDropdown.jsx

```diff
@@ -49,7 +49,7 @@ export default function PastAppointmentsDateDropdown({
           featureStatusImprovement,
         })}
         id="date-dropdown"
-        value={dateRangeIndex}
+        value={dateRangeIndex.toString()}
```

Comment:
Q: why does it need to convert to string?

Reply:
This fixes props warning message about number provided when string was expected.